templates/company/contracts/CompanyTemplate.sol

```diff
@@ -30,48 +24,66 @@ contract CompanyTemplate is BaseTemplate {
         _ensureMiniMeFactoryIsValid(_miniMeFactory);
     }
 
-    function newTokenAndInstance(string _id, address[] _holders, uint256[] _stakes) public {
-        newToken();
-        newInstance(_id, _holders, _stakes);
+    function newTokenAndInstance(
+        string _id, 
+        address[] _holders, 
+        uint256[] _stakes, 
+        string _tokenName, 
+        string _tokenSymbol,
+        uint64 _voteDuration,
+        uint64 _supportRequired,
+        uint64 _minAcceptanceQuorum,
+        uint64 _financePeriod
+    ) 
+        public 
```

Comment:
Not tied to these changes, but can we make all the `newTokenAndInstance` functions external?

Reply:
Done. Note that no changes were made to templates that do not use the newToken/newTokenAndInstance function pair.

Reply:
Probably not yet for this transaction, but the total should have been reduced since we are not storing the finance and agent apps any more